dali/operators/image/crop/crop_mirror_normalize.cc

```diff
@@ -47,7 +47,8 @@ Supported types: ``FLOAT``, ``FLOAT16``, ``INT8``, ``UINT8``.
   .AddOptionalArg("output_layout",
     R"code(Tensor data layout for the output.)code", TensorLayout("CHW"))
   .AddOptionalArg("pad_output",
-    R"code(Determines whether to pad the output to the number of channels as a power of 2).)code", false)
+    R"code(Determines whether to pad the output using ``fill_values`` value to the number
+of channels as a power of 2).)code", false)
```

Comment:
Determines whether to pad the output so that the number of channels is a power of 2. The value used for padding is determined by the ``fill_values`` argument.

Reply:
Done